package dev.shtanko.network.exeption

class NetworkConnectionException(cause: Throwable?) : Exception(cause)
